src-java/ping-topology/ping-storm-topology/src/main/java/org/openkilda/wfm/topology/ping/bolt/OnDemandResultManager.java

```diff
@@ -180,7 +180,14 @@ private YFlowPingResponse buildYFlowPingResponse(Group group) {
                     "Expect exact one Y Flow id in pings group response, got - %s", yFlowIdSet));
         }
 
-        return new YFlowPingResponse(yFlowIdSet.iterator().next(), null, new ArrayList<>(subFlowMap.values()));
+        String error = oneSwitchFlowExists(subFlowMap) ? "One subflow is one-switch flow" : null;
```

Comment:
1. Please, be consistent at least with yourself: you have `sub flow` in FlowFetcher:238 and `subflow` here.
2. Please, update functional test as well, otherwise it will start to fail: YFlowPingSpec:84

Reply:
Sorry, I forgot this. 
Done.